apps/balancer-tools/src/app/apr/(components)/PoolPctVotesInRound.tsx

```diff
@@ -0,0 +1,47 @@
+import invariant from "tiny-invariant";
+
+import { Pool } from "#/lib/balancer/gauges";
+import { DuneGaugeData } from "#/lib/dune";
+import { fetcher } from "#/utils/fetcher";
+
+export default async function PoolPctVotesInRound({
+  roundId,
+  poolId,
+}: {
+  roundId?: string | string[];
+  poolId: string | string[];
+}) {
+  invariant(!Array.isArray(roundId), "roundId cannot be a list");
+  invariant(!Array.isArray(poolId), "poolId cannot be a list");
+
+  const gaugeData = await fetcher<DuneGaugeData[] | { error: string }>(
+    `http://localhost:3000/apr/rounds/${roundId}`,
```

Comment:
shoudn't this be in the .env?

Reply:
good catch! Not in the .env but it should be using the URL host from the request instead of doing this